library/1.0.0/metadata.py

```diff
@@ -0,0 +1,72 @@
+from . import utils
+
+
+def get_header(app_name: str):
+    return f"""# Welcome to TrueNAS SCALE
```

Comment:
You can use `textwrap` here to fix the formatting but i think we should/can probably revisit this once at least we have the functionality in and clean it up appropriately